code/ConvertToStandardPath_SubmarineCable.py

```diff
@@ -11,17 +11,19 @@
 def floatFormatter(number):
     return float("{:.2f}".format(number))
 
+
```

Comment:
In the future, please avoid new commits for the sole purpose of formatting. Ideally format before you commit. This clutters commit history.

Reply:
I will, thanks for reminder.